architecture-decision-records/0018-govkuk-notify-staging-configuration.md

```diff
@@ -0,0 +1,21 @@
+# 18. GovUK Notify Staging Service made Live
+
+**Date:** 2021-04-22
+
+This decision supersedes decision: [0015 GovUK Notify Staging Configuration](0015-govuuk-notify-staging-configuration.md)
```

Comment:
Could you add a link from ADR 15 back to this one, so that we can see that it's been superseded?

Reply:
Good point